test/variable.jl

```diff
@@ -65,6 +65,14 @@ facts("[variable] get and set bounds") do
     @fact getUpper(fixedvar) --> 5
 end
 
+facts("[variable] get and set values") do
+    m = Model()
+    x0 = [1:3]
+    @defVar(m, x[1:3])
+    setValue(x,x0)
+    @fact getValue(x).innerArray --> x0
```

Comment:
Would also be good to add a test for the error case with mismatching dimensions.